_data/tools.yml

```
@@ -1697,3 +1697,15 @@
       url: https://docs.getzingdata.com/docs/setting-up-a-data-source/trino_setup/
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Trino Guru
+  anchor: trino-guru
+  category: add-on
+  logo: /assets/images/logos/gurubase_logo.png
+  logosmall: /assets/images/logos/gurubase_logo_small.png
```

```suggestion
  logosmall: /assets/images/logos/guru-base-small.png
```

Please rename file accordingly